app/controllers/kuroko2/api/job_definitions_controller.rb

```diff
@@ -0,0 +1,81 @@
+class Kuroko2::Api::JobDefinitionsController < Kuroko2::Api::ApplicationController
+  include Garage::RestfulActions
+
+  private
+
+  def require_resources
+    definitions = Kuroko2::JobDefinition.all
+    @resources = definitions.map {|definition| Kuroko2::Api::JobDefinitionResource.new(definition) }
+  end
+
+  def create_resource
+    definition = Kuroko2::JobDefinition.new(definition_params(params))
+    unless definition.api_allowed?
+      raise_not_allowed(definition)
+    end
+    user_ids = admin_id_params(params)
+    definition.admins = Kuroko2::User.active.with(user_ids)
+
+    if definition.save
+      user_ids.each do |user_id|
+        user = Kuroko2::User.active.find(user_id)
+        user.stars.create(job_definition: definition) if user.google_account?
+      end
+      
+      @resource = Kuroko2::Api::JobDefinitionResource.new(definition)
+    else
+      raise HTTP::Forbidden.new("#{definition.name}: #{definition.errors.full_messages.join()}")
+    end
+  end
+
+  def require_resource
+    definition = Kuroko2::JobDefinition.find(params[:id])
+    @resource = Kuroko2::Api::JobDefinitionResource.new(definition)
+  end
+  
+  def update_resource
+    definition = Kuroko2::JobDefinition.find(params[:id])
+    unless definition.api_allowed?
+      raise_not_allowed(definition)
+    end
+    definition.update_attributes(definition_params(params))
+    @resource = Kuroko2::Api::JobDefinitionResource.new(definition)
+  end
+
+  def destroy_resource
+    definition = Kuroko2::JobDefinition.find(params[:id])
+    unless definition.api_allowed?
```

Comment:
ditto https://github.com/cookpad/kuroko2/pull/87/files#r135692164

Reply:
I have fixed it in 55b4da1.